### PR TITLE
add subtask to exports

### PR DESCRIPTION
**Pull request by it-harrison**
## Description
This PR adds a missing export alias to babel.config.json and the platform/forms workspace package.json

## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/forms/exportsFile.js [hunk 1]
```diff
@@ -123,11 +123,22 @@ import {
...
 export default {
+  Subtask,
+  SUBTASK_SESSION_STORAGE,
+  getStoredSubTask,
+  setStoredSubTask,
+  resetStoredSubTask,
   states,
   militaryStates,
   militaryCities,
-  branchCodeLabels,
```
Comment (bkjohnson) [inline src/platform/forms/exportsFile.js:130]: Why is the `branchCodeLabels` export being removed?

**Review by bkjohnson — COMMENTED** [review]
I know that this pattern was established before the PR, but is there a reason for the exportsFile when each individual file is also listed as an export?

File: src/platform/forms/exportsFile.js (hunk 1 above)
Comment (it-harrison) [reply src/platform/forms/exportsFile.js:130]: good catch - I didn't quite finish fixing an earlier bug - fixed in latest commit.

**Comment by it-harrison** [comment]
> I know that this pattern was established before the PR, but is there a reason for the exportsFile when each individual file is also listed as an export?

it is redundant but the idea was to give developers another option for importing

File: src/platform/forms/exportsFile.js (hunk 1 above)
Comment (bkjohnson) [reply src/platform/forms/exportsFile.js:130]: Adding some context for anyone looking at this in the future. The [`branchCodeLabels` import was made in a commit on August 19](https://github.com/department-of-veterans-affairs/vets-website/pull/21977/commits/fa225953cc66d000cbba6d1743ddf553b74ee2df), then the const was [renamed in another PR on August 23](https://github.com/department-of-veterans-affairs/vets-website/pull/21976/files#diff-8fcc2b9b82bc1241bab1450166640c7888f127ff7181f0395aaa30893d8316a4R3).

**Review by bkjohnson — APPROVED** [review]

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**